.github/workflows/coverage.yml

```diff
@@ -0,0 +1,57 @@
+name: Code Coverage
+
+on: [ pull_request ]
+
+jobs:
+  coverage-report:
+    name: Coverage Report
+    runs-on: macos-latest
```

Comment:
@nuxzero can you leave a note on why we currently need `macos-latest` instead of `ubuntu-latest`?